### DISCUSSION TITLE
Hero 11 - Overlay Timestamp

### DISCUSSION THREAD

**Discussion by MethodMan007** [gopro/labs#747] [General]
I've read everything I can find on this issue but I can't figure out what is wrong.  I have a Hero 11 and I'm trying to use the overlay feature to show the time and date.  (https://gopro.github.io/labs/control/overlays/)  
The Hero 11 shows the checkmark on the QR code when I scan it yet nothing ever appears on the video.  I tried multiple different settings on the overlay page that produces the QR code.  I also have tried all the video formats it's supposed to support and I can't get anything to actually show up on the recorded videos. 

Anybody have any suggestions?

**Comment by dnewman-gpsw** [comment]
Share the codes aren't getting success with.  It does work in 4K 24/25/30 and 8-bit, 16x9.  So start there.  You see nothing different on the preview, as it only modifies the recorded media.

**Comment by kouyoumin** [comment]
Possibly related to the [font size issue](https://github.com/gopro/labs/issues/664).
